orchestration/scripts/submit-file-ingest.py

```diff
@@ -0,0 +1,27 @@
+import google.auth
+from google.auth.transport.requests import AuthorizedSession
+from requests.exceptions import HTTPError
+import os
+
+credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
+
+base_url = os.environ["API_URL"]
+dataset_id = os.environ["DATASET_ID"]
+profile_id = os.environ["PROFILE_ID"]
+source_path = os.environ["SOURCE_PATH"]
+target_path = os.environ["TARGET_PATH"]
+
+authed_session = AuthorizedSession(credentials)
+
+headers = {"accept": "application/json",
+           "Content-Type": "application/json"}
```

Comment:
I don't think these are needed (but I guess it doesn't hurt anything)

Reply:
true, forgot to remove from this one